apidump/apidump.go

```diff
@@ -553,11 +559,10 @@ func (a *apidump) Run() error {
 				return errors.Errorf("Cannot automatically rotate sessions when a session name is provided.")
 			}
 		}
-	} else if args.PostmanCollectionID != "" {
+	} else if (args.PostmanCollectionID != "" || args.ServiceID != akid.ServiceID{}) {
 		args.Out.AkitaURI = &akiuri.URI{
-			ObjectType: akiuri.TRACE.Ptr(),
-			// Placeholder name for postman collectionIds
-			ServiceName: "Postman_" + args.PostmanCollectionID,
+			ObjectType:  akiuri.TRACE.Ptr(),
+			ServiceName: a.backendSvcName,
```

Comment:
Create a new `backendSvcName` var since we will be getting the project name from BE in case of `--project`.
Setting this var above in `LookupService()` function

Reply:
**Blocker.** How does this get used? AkitaURIs don't uniquely identify a service anymore, now that service names are no longer unique.

Reply:
Yes, serviceName is not being used now. The only use is in the log message.
https://github.com/akitasoftware/akita-cli/blob/80cebed4d07a9a28ba84cb9c5f2154c484817e6e/apidump/apidump.go#L588

Example: `[INFO] Created new trace on Postman Cloud: akita://apidump-native:trace:ribbon-carp-a9e625e9`

I have added it just to maintain consistency with the old code. 
When using `--collections` we don't have any service name so we use `Postman_random()`, and now since we have serviceName, I am setting it. Also, it can be useful in logs.

The best option is to completely remove the `out` variable and `akita.Uri` concept, which requires more change, which we can do during Akita Sunset.